### ISSUE TITLE
Empty collection and   @ElementCollection

### ISSUE THREAD

**Issue by jfgosselin** [Impetus/kundera#355]
With Cassandra and Kundera 2.7-SNAPSHOT when I persist an entity with an empty @ElementCollection e.g. (empty HashSet for b) :

```
  @Id
  @Column(name = "a")
  private String a;

  @ElementCollection
  @Column(name = "b")
  private Set<String> b;

  @Column(name = "c")
  private String c;
```

I get the following exception :

```
INFO  2013-08-15 09:50:18,776 [main] com.impetus.client.cassandra.CassandraClientBase -  Returning cql query  INSERT INTO "test"("a","b","c") VALUES('myid',,'allo') .
.
.
ERROR 2013-08-15 09:50:18,785 [main] com.impetus.client.cassandra.CassandraClientBase -  Error while persisting record. Caused by: .
InvalidRequestException(why:line 1:60 no viable alternative at input ',')
    at org.apache.cassandra.thrift.Cassandra$execute_cql3_query_result.read(Cassandra.java:37849)
```

**Comment by impetus-opensource-admin** [comment]
I think, empty or null collection instance should be ignored. I will verify and get it fixed.

-Vivek

**Comment by jfgosselin** [comment]
Also a list of string isn't working e.g. :

```
@ElementCollection
@Column(name = "b")
private List<String> b;

 .
 .

CREATE TABLE test (
 a text PRIMARY KEY,
  b list<text>,
   c text
 )
```

The quote are missing in the array :

   ERROR 2013-08-20 10:52:32,021 [main] com.impetus.client.cassandra.pelops.PelopsClient -  Error during persist while executing query  INSERT INTO "test"("a","b","c") VALUES('allo',[1, 2],'c') , Caused by: .
InvalidRequestException(why:Invalid list literal for b: value 1 is not of type text)

**Comment by kkmishra** [comment]
Hi,
Changes pushed in current trunk branch, please verify and let me know if
problem persists.

Thanks
Kuldeep

On Tue, Aug 20, 2013 at 8:36 PM, jfgosselin notifications@github.comwrote:

> Also a list of string isn't working e.g. :
> 
> @ElementCollection
> @Column(name = "b")
> private List<String> b;
> 
>  .
>  .
> 
> CREATE TABLE test (
>  a text PRIMARY KEY,
>   b list<text>,
>    c text
>  )
> 
> The quote are missing in the array :
> 
> ERROR 2013-08-20 10:52:32,021 [main]
> com.impetus.client.cassandra.pelops.PelopsClient - Error during persist
> while executing query INSERT INTO "test"("a","b","c") VALUES('allo',[1,
> 2],'c') , Caused by: .
> InvalidRequestException(why:Invalid list literal for b: value 1 is not of
> type text)
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/impetus-opensource/Kundera/issues/355#issuecomment-22951623
> .

## 

Thanks and Regards
Kuldeep Kumar Mishra
+919540965199

**Comment by mevivs** [comment]
Fixed and releasing today with 2.6.1

-Vivek
